config/application.yml.default

```diff
@@ -174,7 +174,6 @@ in_person_outage_message_enabled: false
 in_person_proofing_enabled: false
 in_person_proofing_enforce_tmx: false
 in_person_proofing_opt_in_enabled: false
-in_person_public_address_search_enabled: false
```

Comment:
You may need to make 2 separate pull requests because of the 50/50 state, see documentation [here](https://handbook.login.gov/articles/manage-50-50-state.html#test-in-the-5050-state). 

During the deployment (for about 20 minutes)- user might be directed to the original or the new version of `app/controllers/idv/in_person/public/usps_locations_controller.rb`, for example. Since the original version references the variable- it will cause an error because it has been deleted.

I think this ticket is 3 steps:
1. PR#1 Delete conditionals and specs, as you did.
2. PR#2 Delete variables in `config/application.yml.default` and `lib/identity_config.rb`
3. Delete s3-secrets (no pull request)

Please read about the 50/50 state. It would be nice to get other @18F/identity-joy to agree or disagree before you start so I don't mistakenly have you do extra work if I am wrong.

Reply:
I think I disagree. My thinking is that when the application starts it grabs a snapshot the environment and it won't change, even if the environment configurations change, until that application restarts. So, the old application running should have what it needs run and the new application code does not require it run. Therefore, there should be no 50/50 state conflict during the deployment unless the environment has this feature turned off. 

We should make sure that the flag is set to true in production... so we can avoid the on/off between old code and new code. I don't think we have to worry about it in lower environments.

Reply:
I think you're onto something since the ticket said this may have to be phased. But ill let others weigh in too!

Reply:
Confirmed its true in prod

Reply:
Oh, I am silly. I forget that even though these config values are deleted here- they still exist because in s3. I wrote step 3- they are still there. I am sorry @WilliamBirdsall. Thanks for reviewing @shanechesnutt-ft.  Resume full steam as is  🚂

Reply:
No need to apologize! It was a valid consideration :)

Will mark this for review, read up a bit on changing s3 vars and then merge it in today.